Containerfile.mass-rebuild

```diff
@@ -1,6 +1,12 @@
-FROM registry.fedoraproject.org/fedora:41
-
-ENV LLVM_SYSROOT=/opt/llvm \
+ARG arch=x86_64
+ARG base_image=registry.fedoraproject.org/fedora:41-${arch}
```

Comment:
AFAIU, if `--platform` is used, you don't need to specify the architecture here.